cmd/root.go

```diff
@@ -329,9 +340,26 @@ func initKonnectConfig() error {
 		password = strings.TrimRight(password, "\n")
 	}
 
+	token := viper.GetString("konnect-token")
+	tokenFile := viper.GetString("konnect-token-file")
+	// read from token file only if token is not supplied using an
+	// environment variable or flag
+	if token == "" && tokenFile != "" {
+		fileContent, err := ioutil.ReadFile(tokenFile)
+		if err != nil {
+			return fmt.Errorf("read file %q: %w", tokenFile, err)
+		}
+		if len(fileContent) == 0 {
+			return fmt.Errorf("file %q: empty", tokenFile)
+		}
+		token = string(fileContent)
+		token = strings.TrimRight(token, "\n")
+	}
```

Comment:
Add a check to make sure the user supplies either a token or a (email and password).
Try using [this](https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsMutuallyExclusive) to minimize if/else statements.